doc/classes/@GDScript.xml

```diff
@@ -415,6 +415,21 @@
 			<return type="Array">
 			</return>
 			<description>
+				Returns an array of dictionaries representing the current call stack.
+				[codeblock]
+				func _ready():
+					foo()
```

Comment:
Code in code blocks should be indented with 4 spaces.

Reply:
Fixed